specs/opcode/16AND.md

```diff
@@ -0,0 +1,50 @@
+# AND op code
+
+## Procedure  
+
+A stack initalize empty with stack pointer to 1024, pop operation can only happen when stack is not empty, and it will increase by 1 of stack pointer.  
+
+The And value will be dropped directly without no any more checking & utilizing.
+
+## Constraints
+
+1. opId = OpcodeId(0x16u8)
+2. state transition:  
+    - gc + 3
+    - stack_pointer + 1
+    - pc + 1
+    - gas + 3
+
+
+## Exceptions
+
+1. stack underflow: when stack is empty
+2. gas out: remaining gas is not enough   
+
+```python
+class Stack():
```

Comment:
Is this code needed here? Also, I am not sure about pop and push implementation? There is a code for Stack in 50POP.md which has the pop and push implementation as it's in Geth. 

BTW @ChihChengLiang : should we move code for Stack in 50POP.md into src folder?

Reply:
Yes, I think that's a good idea.